server/command.go

```diff
@@ -248,8 +255,33 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 	return false, nil
 }
 
+func (p *Plugin) runSettingsCommand(args []string, extra *model.CommandArgs) (bool, error) {
+	if args[0] == "summary" {
+		var responseMessage string
+		var err error
+
+		if args[1] == "on" {
+			err = p.saveReminderPreference(extra.UserId, true)
+		} else if args[1] == "off" {
+			err = p.saveReminderPreference(extra.UserId, false)
+		}
```

Comment:
Using a `switch` here would make the code cleaner and would make it easy to add a `default` path.

Reply:
Changed to switch model.